Cargo.toml

```
@@ -73,7 +73,7 @@ name = "api_trait_implementation"
 required-features = ["telegram-trait"]
 
 [dependencies]
-bon = "2.2.0"
+bon = "3.0.2"
```

Guess 3.0.0 would be the minimum. The fix included with 3.0.1 is not relevant in frankenstein (currently?) and 3.0.2 fixes a lint warning on nightly only but still works fine without.

```suggestion
bon = "3.0.0"
```